lalrpop-test/src/optional_semicolon.rs

```diff
@@ -0,0 +1,710 @@
+// auto-generated: "lalrpop 0.15.0"
```

Comment:
This file should be .gitignored (or with #353 it may be put in OUT_DIR instead)

Reply:
I think I'll just write a unit-tests instead. I think I found some in the `grammar` directory.